doc/dev/cputrace.rst

```diff
@@ -0,0 +1,312 @@
+========
+CpuTrace
+========
+
+CpuTrace is a tool to measure hardware cost of program execution.
+Its intended user is a developer; either when he writes new code and
+cannot decide which algorithm is better, or when he has an idea for
+a possible performance optimization, but wants to make sure.
```

Comment:
CpuTrace is a developer tool that measures the CPU cost of execution.  It is useful when deciding between algorithms for new code and for validating performance  enhancements.  

The core of this rephrasing is to be inclusive by not  referring to the dev as “he”.